plugins/org.eclipse.n4js.ide/src/org/eclipse/n4js/ide/server/util/ServerIncidentLogger.java

```diff
@@ -43,8 +43,10 @@
 
 	/** Path of the folder where the log files are placed, relative to the user's home directory. */
 	public static final Path SERVER_INCIDENTS_FOLDER = Path.of(".n4js", "server-incidents");
-	/** Base name of the log file, will be amended by a time stamp. */
-	public static final String BASE_FILE_NAME = "server-incident_.log";
+	/** Base name of the log file. */
+	public static final String DEFAULT_BASE_FILE_NAME = "server-incident";
+	/** Suffix and extension of the log file name. The suffix will be amended by a time stamp. */
+	public static final String BASE_FILE_NAME_EXTENSION = "_.log";
```

Comment:
minor: this constant could be renamed to `FILE_EXTENSION` and the `_` could be removed from the value here and appended to the base name (hard coded) below in `#doCreateFile()` when the entire file name is built.